showcase/src/app/demo-ui/components/table-with-items-per-page-selector/table-with-items-per-page-selector.component.html

```diff
@@ -0,0 +1,14 @@
+<stark-table
+	[data]="data"
+	[columnProperties]="columns"
+	[filter]="filter"
+	[orderProperties]="order"
+	[paginationConfig]="pagination"
+	[tableRowActions]="tableRowActions"
+	showRowIndex
```

Comment:
Is the `showRowIndex` necessary here ?
I have the feeling it's useless in this case 😕